website/docs/d/is_security_groups.html.markdown

```diff
@@ -0,0 +1,115 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_security_groups"
+description: |-
+  Get information about SecurityGroupCollection
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm_is_security_groups
+
+Provides a read-only data source for SecurityGroupCollection. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```

Comment:
Refer the resource of security group rule and add the reference here

Reply:
done.

Reply:
done.